src/utils/loaders/google_map_loader.js

```diff
@@ -60,28 +61,29 @@ export default (bootstrapURLKeys, heatmapLibrary) => {
       if (Object.keys(bootstrapURLKeys).indexOf('callback') > -1) {
         const message = `"callback" key in bootstrapURLKeys is not allowed,
                           use onGoogleApiLoaded property instead`;
+        // eslint-disable-next-line no-console
         console.error(message);
         throw new Error(message);
       }
     }
 
-    let queryString = Object.keys(bootstrapURLKeys).reduce(
+    let params = Object.keys(bootstrapURLKeys).reduce(
```

Comment:
The word `params` makes more sense than `queryString`, as they're actually _params_